docs/DeveloperGuide.md

```diff
@@ -2,28 +2,454 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
```

Comment:
Curly bracket comments can be omitted

Reply:
This line can be used to describe the items below, rather than saying, in a literal manner, what the point of that segment is.
eg.: Listed below are the third-party sources used in this project. Points are hyperlinked to their original source.